07-react.js/react-boilerplate-master/components/title-component.js

```diff
@@ -0,0 +1,12 @@
+// VENDOR LIBS
```

Comment:
Not sure if this is an useful component (since it's just returning an hardcoded title).